### PR TITLE
VMMS operations are blocked node gets stuck in deletion and creation after deleting a VMSS

### PR DESCRIPTION
**Pull request by sedwardstx**

### CODE REVIEW

File: Known_Issues/All VMMS operations are blocked on Silver or higher durability.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# VMMS operations are blocked node gets stuck in deletion and creation after deleting a VMSS
+
+## Problem
+- All VMSS operations are blocked on Silver or higher durability Scaleset after trying to delete a node from a VMMS
```
Comment (craftyhouse) [inline Known_Issues/All VMMS operations are blocked on Silver or higher durability.md:4]: I would suggest linking to the public docs that talk about what durability is https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-cluster-capacity#durability-characteristics-of-the-cluster 

suggestion "silver or higher durability Service Fabric VMs..."

typo at end VMMS - > VMSS

File: Known_Issues/All VMMS operations are blocked on Silver or higher durability.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
+# VMMS operations are blocked node gets stuck in deletion and creation after deleting a VMSS
+
+## Problem
+- All VMSS operations are blocked on Silver or higher durability Scaleset after trying to delete a node from a VMMS 
+- The node being deleted (seed node) is stuck in Disabling state
```
Comment (craftyhouse) [inline Known_Issues/All VMMS operations are blocked on Silver or higher durability.md:5]: suggestion: expand what seed nodes are (seed node which are used by the Service Fabric cluster services)
or
The node being deleted is a seed node, which are used by the Service Fabric cluster services, and is stuck in Disabling state

File: Known_Issues/All VMMS operations are blocked on Silver or higher durability.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+## Symptoms
+- One of the seed nodes is stuck disabling with Safety Check - **EnsureSeedNodeQuorum**
+- All operation started after the delete operation will be blocked until the deletion completes, including adding new VMs
```
Comment (craftyhouse) [inline Known_Issues/All VMMS operations are blocked on Silver or higher durability.md:9]: Suggestion: expand ", including adding new VMs" to include more examples
scaling operations, cancellation tasks, auto os upgrades, and other operational jobs.

File: Known_Issues/All VMMS operations are blocked on Silver or higher durability.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+**Customer has to determine the root cause of why the VM is being deleted**
+
+- The VMSS operation that resulted in the specific instance is being deleted / deallocated - Audit logs for VMSS in Portal should have the information  
+- Most common reasons  
+  - Customer initiated delete / deallocation from portal or CLI
+    - normally to address some other issue and as a result put the cluster into worse state
+  - Auto scale settings where min is < number of nodes required for reliability level
+  - Some automation run by customer that is initiating the delete
+
+**NOTE:** Deallocate operations on instances in scaleset associated with SF is not supported.
```
Comment (craftyhouse) [inline Known_Issues/All VMMS operations are blocked on Silver or higher durability.md:30]: can we link to docs explaining the supported path or at least mention adjusting the VMSS scaleset?

File: Known_Issues/All VMMS operations are blocked on Silver or higher durability.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+## Example ##
+
+ . | .
+----------|----------
+Current Reliability Level | Silver
```
Comment (craftyhouse) [inline Known_Issues/All VMMS operations are blocked on Silver or higher durability.md:36]: Should we add a line for Durability being silver or higher to make it clear why MR stopped the bad operation?

File: Known_Issues/All VMMS operations are blocked on Silver or higher durability.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+Seed nodes required for current reliability level | 5
+Current number of nodes of Primary node type | 5
+Does removing a node violate reliability level | Yes
+
+## Mitigation ##
+
+**The following is the only safe mitigation**
+
+**NOTE:** Mitigation time will take at a minimum 60-90 minutes. Depending on the configuration of SF cluster upgrade settings it can take significantly longer.  Please open a support ticket through the Azure Portal for assistance.
```
Comment (craftyhouse) [inline Known_Issues/All VMMS operations are blocked on Silver or higher durability.md:45]: Suggestion, slight tweak as we are pushing for self-service?

>Please open a support ticket through the Azure Portal for assistance.
Please open a support ticket through the Azure Portal if you need assistance. (link to support page?)

File: Known_Issues/All VMMS operations are blocked on Silver or higher durability.md (hunk 1 above)
Comment (sedwardstx) [reply Known_Issues/All VMMS operations are blocked on Silver or higher durability.md:4]: fixed

File: Known_Issues/All VMMS operations are blocked on Silver or higher durability.md (hunk 5 above)
Comment (sedwardstx) [reply Known_Issues/All VMMS operations are blocked on Silver or higher durability.md:36]: I added a more extensive table of examples for Silver, Gold, Platinum reliability levels. lmk what you think

**Pull request merged**